sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -399,7 +408,7 @@ export class ManagementClient extends LinkEntity {
       };
       request.application_properties![Constants.trackingId] = generate_uuid();
       // TODO: Update to use constant from AMQP common
-      request.application_properties!["associated-link-name"] = associatedLinkName;
+      request.application_properties![Constants.associatedLinkName] = associatedLinkName;
```

Comment:
Yes, I had excluded it on purpose because of the conversations around value of it not making much difference to service side. 
But I think it does. Sure, can add in the if checks.